src/BuiltInTools/dotnet-watch/DotNetWatchContext.cs

```diff
@@ -26,7 +26,7 @@ public class DotNetWatchContext
 
         public BrowserRefreshServer? BrowserRefreshServer { get; set; }
 
-        public LaunchSettingsProfile DefaultLaunchSettingsProfile { get; init; } = default!;
```

Comment:
Is this API callable other ways and would be considered a BC for anyone else?

Reply:
I don't believe we consider changes to public APIs in tools like this a breaking change. Certainly we don't have any of the usual public API tracking infrastructure in place for the tools that I know of. The biggest concern would be if something in VS is loading the tool assembly directly but I'd hope we're not doing that as it makes the dependency extremely unclear (from this repo's POV).

Reply:
This is only used by the command-line. We don't publish this as a package, so we should be good. In general, anything that we share with Visual Studio has an explicit agreement that defines it.

@BillHiebert can confirm here.